react/typings/globals.d.ts

```diff
@@ -1,11 +1,3 @@
-interface OrderForm {
```

Comment:
I think we can remove the remaining interfaces of this file as well. Is it too much work to change the code to import them from `checkout-graphql`?

Reply:
hmmm, I don't think so. I'll try that

Reply:
actually, these types left in this file weren't used anywhere! :smile: